Userland/Libraries/LibPDF/Fonts/Type1FontProgram.cpp

```diff
@@ -500,6 +500,7 @@ PDFErrorOr<Type1FontProgram::Glyph> Type1FontProgram::parse_glyph(ReadonlyBytes
                 case Hflex1:
                 case Flex1:
                     // TODO: implement these
+                    dbgln_if(PDF_DEBUG, "Unimplemented flex: 12 {}", data[i]);
```

Comment:
Maybe a dumb question, but just in case it was unintentional - what does the `12` signify here?

Reply:
It's an extended 2-byte command. 2-byte commands have the first byte set to 12.